examples/graphql-returning-record-values/graphql_returning_record_values.md

```diff
@@ -1,10 +1,10 @@
 # GraphQL service - Record as output object
 
-In a Ballerina GraphQL service, a record type can be used to represent an `OBJECT` type in a GraphQL schema. The type name and the field names are mapped one-to-one from the Ballerina to GraphQL. Types are generated by recursively analyzing the types that are used in a Ballerina GraphQL service.
+In a Ballerina GraphQL service, a record type can be used to represent an `OBJECT` type in a GraphQL schema. The type name and the field names are mapped one-to-one from the Ballerina to GraphQL. Types are generated by recursively analyzing the types that are used in a Ballerina GraphQL service. Use a record type to represent an `OBJECT` type only when all fields of that object type do not have any input arguments.
 
-This example shows a GraphQL endpoint, which has a field `profile` of type `Profile!` in the root `Query` type. A GraphQL client can query on this service to retrieve specific fields or subfields of the `Profile` object.
+This example shows a GraphQL endpoint, which has a field `profile` of type `Profile!` in the root `Query` type. A GraphQL client can query this service to retrieve specific fields or subfields of the `Profile` object.
 
->**Note:** Although both the record and service types can be used to represent an Object type, using a record type as an Object has limitations. For example, a field represented as a record field can not have an input argument, as opposed to a field represented using a resource method in a service class.
+>**Note:** A service type can also be used to represent an `OBJECT` type when a field in the `OBJECT` type as input arguments.
```

Comment:
We can remove this note since we have already stated this in the description.

Reply:
Updated